projects/core/components/expand/expand.component.ts

```diff
@@ -110,7 +110,7 @@ export class TuiExpandComponent {
     }
 
     get contentVisible(): boolean {
-        return this.expanded || this.state !== State.Idle;
+        return this.expanded ?? this.state !== State.Idle;
```

Comment:
This is incorrect. Content is visible if expanded is true or state is not Idle. Now it is not visible when expanded is false, regardless of state. Please be careful, it's hard to review this many changes.